tests/shared/robot.spec.js

```diff
@@ -131,16 +131,16 @@ describe('Shared: Robot', function() {
         data: {
           message: 'foo',
           code: 'MYERROR',
-          stacktrace: error.stack
+          stack: error.stack
         }
       });
     });
 
     it('should handle a string error', function() {
```

Comment:
...except `robot.buildErrorMessage` is _only_ called from `robot.printError`, and you already changed `robot.printError` to only accepting `Error` objects?

Oh, you've changed `robot.printError` back to `{(Error|String)}` :-/
Looks like the only places `robot.printError` is used are https://github.com/resin-io/etcher/blob/master/lib/cli/options.js#L96 and https://github.com/resin-io/etcher/blob/master/lib/cli/etcher.js#L124 so I wonder if it'd be worth keeping `robot.printError` as only accepting `Error`, and then using `errors.createError` in https://github.com/resin-io/etcher/blob/master/lib/cli/etcher.js#L124 to create a proper Error object if we only get a string back?

Or am I just overcomplicating things? :-S